src/cascadia/TerminalSettingsModel/TerminalSettings.h

```diff
@@ -97,6 +97,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         INHERITABLE_SETTING(Model::TerminalSettings, bool, AllowVtChecksumReport, false);
         INHERITABLE_SETTING(Model::TerminalSettings, bool, TrimBlockSelection, true);
         INHERITABLE_SETTING(Model::TerminalSettings, bool, DetectURLs, true);
+        INHERITABLE_SETTING(Model::TerminalSettings, bool, AllowVtClipboardWrite, false);

```

Comment:
```suggestion
        INHERITABLE_SETTING(Model::TerminalSettings, bool, AllowVtClipboardWrite, true);
```
This is set to `true` everywhere else

Reply:
HOW DID I MISS THAT UGH THANKS